terminal/answers/samcrider.md

```diff
@@ -0,0 +1,36 @@
+## QUESTIONS
+
+### What are the top 20 commands for mac Command Line Interface (CLI)?
```

Comment:
nice! I'd also recommend grep, grok, find, echo, and cat which are all fairly common commands